src/components/Header/header.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import styled from "styled-components";
+
+const StyledHeaderDiv = styled.div`
+  width: 100vw;
+  height: 60px;
+  background: rgba(214, 245, 245, 0.5);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  i {
+    color: var(--header);
+    margin-right: 20px;
+    margin-left: 10px;
+    font-size: 20px;
+    cursor: pointer;
+  }
+`;
+
+const StyledImgWrapperDiv = styled.div`
+  line-height: 60px;
+
+  img {
+    image-rendering: pixelated;
+    vertical-align: middle;
+    padding: 0 20px;
+  }
+`;
+
+function Header() {
+  return (
+    <StyledHeaderDiv>
+      <StyledImgWrapperDiv>
+        <img src="images/logo.png" alt="logo" />
+      </StyledImgWrapperDiv>
+      <div>
+        <i className="fas fa-envelope"></i>
+        <i className="fas fa-user-plus"></i>
+        <i className="fas fa-user-friends"></i>
+        <i className="fas fa-store"></i>
+        <i className="fas fa-sign-out-alt"></i>
+      </div>
```

Comment:
```suggestion
      <StyledMenuWrapperDiv>
        <i className="fas fa-envelope"></i>
        <i className="fas fa-user-plus"></i>
        <i className="fas fa-user-friends"></i>
        <i className="fas fa-store"></i>
        <i className="fas fa-sign-out-alt"></i>
      </StyledMenuWrapperDiv>
```

위에 이미지는 styled로 감싸주었는데, 아래는 그냥 div를 써주셔서, styled로 통일 시키는게 깔끔해 보이긴 하는데 어떨까요?
개인적으로는 12-18번 라인의 스타일 코드도 아래쪽 div가 가지면 조금 더 이해가 편할 것 같습니다.

Reply:
저는 불필요한 styledcomponent 는 최대한 피하려고 했었는데요! 지원님 말씀도 일리가 있는 것 같아요 🙂  
i tag에 스타일을 주는 상황이니까 말씀대로 수정해보도록 할게요 😇